src/Chapter2.hs

```diff
@@ -620,7 +648,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven (x:y:xs) = x : takeEven xs
```

Comment:
Great pattern Matching skills! 💪🏼 

Note, that you don't use the middle element in the implementation. So you can use a wildcard instead of giving the argument name 🙂 
```suggestion
takeEven (x:_:xs) = x : takeEven xs
```